research-on-biomedical-engineering.csl

```diff
@@ -85,10 +81,14 @@
     </choose>
   </macro>
   <macro name="access">
-    <text variable="PMID" prefix="PMid:" suffix=". "/>
+    <choose>
+      <if match="any" variable="PMID">
+        <text variable="PMID" prefix="PMid:" suffix=". "/>
+      </if>
+    </choose>
```

Comment:
@damnation333, same here. Could you incorporate @adam3smith's changes from https://github.com/citation-style-language/styles/pull/2538/commits/7d1a4bfc638cdeef436d2b2da4c642af64ed2647 ?